<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware_update</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Conn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>33</width>
      <height>29</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/refresh.png</normaloff>Resources/refresh.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ports">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>43</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prog">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Progr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ind_conn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_axis">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>43</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Axis:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_axis">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>29</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>22</height>
     </size>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>503</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file_path">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>70</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButt_file">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../zzz/velograph_II_update/velograph_II_update/velograph_ii_update.qrc"/>
 </resources>
 <connections/>
</ui>
